bin/run

```diff
@@ -1,14 +1,19 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -eu
 
+if [ -z "${CONTAINER_ENGINE:+x}" ]; then
+	CONTAINER_ENGINE=docker
+fi
+
 W=/workspace
+
 # Create container
-C=$(docker container create --rm -w $W ghcr.io/codewars/coq:latest sh -c "coqc Solution.v && coqc -I /opt/coq_codewars/src -Q /opt/coq_codewars/theories CW SolutionTest.v")
+C=$($CONTAINER_ENGINE container create --rm -w $W ghcr.io/codewars/coq:latest sh -c "coqc Solution.v && coqc -I /opt/coq_codewars/src -R /opt/coq_codewars/theories CW SolutionTest.v")
```

Comment:
Why was `-Q` changed to `-R`? `/opt/coq_codewars/theories` doesn't have subdirectories.

Reply:
I'm not sure :sweat_smile: 

Let me revert the change and see if it runs properly.